NewForce-Cohort-4/tabloid-full-stack-team-magenta
# Tf create tag

# Description
​As an admin I would like to be able to create a new Tag so that authors may better classify their posts. Given an admin is viewing all tags they should be presented with a button allowing them to create a new tag. Once they have navigated to that page they should be afforded the ability to create and save a new name for a tag that will be added to the list of tags available. 
​
## Type of change

#  Testing Instructions 

1. Pull down and checkout my branch tf-create-tag
2. Start the application in VS and run the program
3. When the application loads you will also need to open the client-side portion of the project and run `npm start` from the terminal
4.  use foo@bar.com and password: password to log in to the application 
5. Once logged in navigate to the `tags` tab and click the link shown
6. At this point an Admin should see a button at the bottom of the list to create a new tag that should be clicked
7. From the new route the Admin can add a tag which should save to the database and then take them back to the view of all tags showing the newly inserted one.. ensuring the ticket was completed

​
# Checklist:
​
